index.js

```diff
@@ -73,9 +76,16 @@ function getIt (options, done) {
           var filename = path.basename(resource.line);
 
           console.log('Start fetching', resource.line);
-
+          
```

Comment:
Extra white space. Depending on the editor you are using there are ways to specify that it should remove all spaces at the end of a line.